locust/web.py

```diff
@@ -164,7 +164,8 @@ def swarm() -> Response:
                             user_classes[user_class_name] = user_class_object
 
                 else:
-                    user_classes = self.environment.available_user_classes
+                    user_classes = {key: value for (key, value) in self.environment.available_user_classes.items if
```

Comment:
I tried out this change and got
```
File "/Users/michaelnester/git/locust/locust/web.py", line 167, in swarm
  user_classes = {
TypeError: 'builtin_function_or_method' object is not iterable
```

I think it needs to be `self.environment.available_user_classes.items()`

Reply:
yes, sorry. My typo error. I will put the unit test code soon